src/tests/functional/plugin/shared/src/low_precision_transformations/multiply_with_one_parent_transformation.cpp

```diff
@@ -29,19 +29,21 @@ std::string MultiplyWithOneParentTransformation::getTestCaseName(const testing::
 }
 
 void MultiplyWithOneParentTransformation::SetUp() {
-    threshold = 0.01f;
+    rel_threshold = 0.01f;
 
     ngraph::element::Type netPrecision;
```

Comment:
Replace `ngraph` types with `ov` equivalents.
Check other tests and remove `ngraph` namespace usage.